lib/Minz/Migrator.php

```diff
@@ -180,9 +178,9 @@ public function addMigration($version, $callback) {
 	 *
 	 * @see https://www.php.net/manual/en/function.strnatcmp.php
 	 *
-	 * @return array
+	 * @return array<string, callable>
```

Comment:
```suggestion
	 * @return array<string,callable>
```

My syntax highlight does not work well when there are spaces in types :-)